src/System.IdentityModel.Tokens.Jwt/LogMessages.cs

```diff
@@ -50,7 +50,6 @@ internal static class LogMessages
         internal const string IDX12713 = "IDX12713: Creating actor value using actor.BootstrapContext(as string)";
         internal const string IDX12714 = "IDX12714: Creating actor value using actor.BootstrapContext.rawData";
         internal const string IDX12715 = "IDX12715: Creating actor value by writing the JwtSecurityToken created from actor.BootstrapContext";
```

Comment:
I've seen something, but maybe it was just in MISE, where we comment out the removed log message, instead of deleting it, to have a record of it in the code. At least the #'s used.